libraries/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -283,6 +285,17 @@
  */
 #define MQTT_INTERNAL_FLAG_BLOCK_ON_SEND                       ( 0x80000000 )
 
+/**
+ * @brief When calling _IotMqtt_RemoveSubscriptionByPacket(), use this value
```

Comment:
_IotMqtt_RemoveSubscriptionByPacket is a documented function and should be referred to as `#_IotMqtt_RemoveSubscriptionByPacket`

Reply:
Changed